README.md

```diff
@@ -39,8 +39,16 @@ npm install -g node-html-to-image-cli
 
 ## Usage
 
+Without handlebars template
+
+```sh
+npx node-html-to-image-cli examples/simple.html ./image.png
+```
+
+With additional data passed to the template
+
 ```sh
-npx node-html-to-image-cli ./index.html ./image.png
+npx node-html-to-image-cli examples/handlebars.html ./image.png --content ./examples/handlebars.json
```

Comment:
When users install the cli they don't have the examples folder. I prefer it doesn't appear in the usage examples.

Reply:
The same could be said of `index.html`in the user's current directory. 

We can either inline the examples or  mention that the examples folder is located in the repository

Reply:
A mention with a link to the examples folder would be nice 👍

Reply:
I can add it after if you want @jtanguy ?